homeassistant/components/motionmount/__init__.py

```diff
@@ -0,0 +1,53 @@
+"""The Vogel's MotionMount integration."""
+from __future__ import annotations
+
+import motionmount
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST, CONF_PORT, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .const import DOMAIN
+from .coordinator import MotionMountCoordinator
+
+PLATFORMS: list[Platform] = [
+    Platform.NUMBER,
+]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Vogel's MotionMount from a config entry."""
+
+    hass.data.setdefault(DOMAIN, {})
+
+    # Create API instance
+    mm = motionmount.MotionMount(entry.data[CONF_HOST], entry.data[CONF_PORT])
+    coordinator = MotionMountCoordinator(hass, mm)
+    mm.add_listener(coordinator.motionmount_callback)
+
+    # Validate the API connection
+    try:
+        await mm.connect()
+        await coordinator.async_config_entry_first_refresh()
```

Comment:
The coordinator already catches their own exception, can be put outside of the try block